src/js/stores/__tests__/DCOSStore-test.js

```diff
@@ -320,7 +320,7 @@ describe("DCOSStore", function() {
 
     it("updates the service tree", function() {
       expect(DCOSStore.serviceTree.getItems()[0].getStatus()).toEqual(
-        "Recovering"
+        "Deploying (Delayed)"
```

Comment:
These were updated, because statuses that used to be "Recovering" are now "Deploying (Delayed)".